igvc_platform/src/imu/YostLabDriver.cpp

```diff
@@ -176,12 +176,12 @@ void YostLabDriver::run()
   setAndCheckIMUSettings();
 
   // Performs auto-gyroscope calibration. Sensor should remain still while samples are taken.
-  this->startGyroCalibration();
+  startGyroCalibration();
 
   // commit settings and start streaming
-  this->flush();
-  this->serialWriteString(SET_STREAMING_TIMING_5_MS);
-  this->serialWriteString(START_STREAMING);
+  flush();
+  serialWriteString(SET_STREAMING_TIMING_5_MS);
+  serialWriteString(START_STREAMING);
 
   ros::Rate loop_rate(20);  // 20Hz
```

Comment:
I just noticed this - we probably don't want a hardcoded rate, and we probably want to get these measurements at a much higher rate than 20Hz. Make this a parameter.